### ISSUE TITLE
Bar Chart: Any way to add a bit of spring?

### ISSUE THREAD

**Issue by talkaboutdesign** [danielgindi/ios-charts#892]
When dragging chart, is there a way to enable a bit of spring action? Be able to drag a bit further then the end, and then snap back. Similar to TableView?

I am using a Bar Chart

**Comment by liuxuan30** [comment]
good idea. We have animator class and ChartViewPortJob classes, I think it can be done there. We could use it, or we can add some other stuff like UIDynamic?

**Comment by danielgindi** [comment]
Bounce on the edges is not supported yet - currently the viewport limits the chart to not go over the edge, but that can be changed for specific cases like animating...

**Comment by eirikvaa** [comment]
Would love this - is it supported now?
